docs/BuildFromSource.md

```diff
@@ -59,10 +59,20 @@ The steps you follow next depend on your preferred development environment:
     ./restore.cmd
     ```
 
-1. You'll typically focus on one project in the repo. You can use the `startvs.cmd` command to launch Visual Studio in a particular project area. For example, to launch Visual Studio in the `Components` project:
+1. After the `restore` script finishes, activate the locally installed .NET by running the following command.
```

Comment:
I don't think you need this for either `startvs.cmd` or `build.cmd`?  At least, I never use it.

Reply:
I am not sure either, but would Visual Studio pick the correct dotnet.exe without it?

Also, do you know in which context it is necessary then? Would this context follow the steps in this file so is it better to keep it there?

Reply:
Yes, startvs sets environment variables to make that work.

I think you would use activate if you wanted to use the dotnet cli in the repo.

Reply:
Isn't `build.cmd` invoking it (dotnet.exe)?

Reply:
I assume it sets whatever environment variables it needs.  If only there were some way to find out...

Reply:
I found a way to find out...

Then removed the step